src/tests/mocks/reentrancy_mock.cairo

```diff
@@ -16,58 +17,59 @@ trait IReentrancyMock<TState> {
 
 #[starknet::contract]
 mod ReentrancyMock {
-    use openzeppelin::security::reentrancyguard::ReentrancyGuard;
+    use openzeppelin::security::reentrancyguard::ReentrancyGuard as reentrancy_guard_comp;
```

Comment:
Not sure if we should go with `_comp` or `_component` but I think we should keep consistency. I understand this name is large, but src5_comp is maybe not the best name? Not strong defense, just pushing for consisteny.

Reply:
That's fair. No strong opinions on this. We can use the full `_component` for the sake of readability